kalite/distributed/static/js/distributed/exercises.js

```diff
@@ -222,7 +222,12 @@ window.AttemptLogCollection = Backbone.Collection.extend({
     },
 
     url: function() {
-        return "/api/attemptlog/?" + $.param(this.filters, true);
+        if (window.statusModel.get("is_logged_in")) {
+            return "/api/attemptlog/?" + $.param(this.filters, true);
+        }
+        else {
+            window.location = "/securesync/login";
+        }
```

Comment:
We should find a more general-purpose way to do this (detect if a logout has happened, and redirect), rather than just having it for attemptlogs. I've opened an issue for this at #3040.